service/controller/v13/key/key.go

```diff
@@ -199,6 +201,14 @@ func EC2ServiceDomain(customObject v1alpha1.AWSConfig) string {
 	return domain
 }
 
+func BaseDomain(customObject v1alpha1.AWSConfig) string {
+	// TODO remove other zones and make it a BaseDomain in the CR.
+	// CloudFormation creates a separate HostedZone with the same name.
+	// Probably the easiest way for now is to just allow single domain for
+	// everything which we do now.
```

Comment:
Can we track this in the board?

Reply:
I have todo here https://github.com/giantswarm/giantswarm/issues/3314
If it happens to be too much work, I'll convert that to be an issue.